shared/chat/conversation/header.native.js

```diff
@@ -19,7 +19,7 @@ const Header = ({muted, onBack, onOpenFolder, onShowProfile, onToggleSidePanel,
         onUsernameClicked={onShowProfile} />
       {muted && <Icon type='iconfont-shh' style={{...styleCenter, ...styleLeft}} />}
     </Box>
-    <Icon type={sidePanelOpen ? 'iconfont-close' : 'iconfont-info'} style={{...styleLeft, flexShrink: 0, marginTop: 4}} onClick={onToggleSidePanel} />
+    <Icon type={sidePanelOpen ? 'iconfont-close' : 'iconfont-info'} style={{...styleLeft, flexShrink: 0, padding: 4}} onClick={onToggleSidePanel} />
```

Comment:
yeah but this was really eyeball. i updated it. Reflecting on this i don't think the global margins is that useful outside of the context of actual margins as if we update it it'll push random shit around everywhere and not actually buy us anything